proxy/src/http/transaction.rs

```diff
@@ -207,7 +207,6 @@ impl ToDocumentedType for ListInput {
     }
 }
 
-#[allow(clippy::result_unwrap_used)]
```

Comment:
It should be okay to continue to use unwraps in tests.

Reply:
Right, I think I removed this by accident.